samples/java/semantickernel-samples/pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Unfortunately cannot have changes outside of /java, we can ignore the sample that is at /samples/java for now, no need to keep that up to date

Reply:
I will remove the file then. Should I remove the `Build semantic-kernel samples` step since the path doesn't exist on the `experimental-java` branch?

Reply:
building the samples that are in `java/samples` on the experimental-java branch is fine for now we can just ignore the `/samples` directory